data/scripts/lib/register_monster_type.lua

```diff
@@ -502,7 +502,7 @@ registerMonsterType.defenses = function(mtype, mask)
 					if defense.target then
 						spell:setNeedTarget(defense.target)
 					end
-					if attack.direction then
+					if defense.direction then
 						spell:setNeedDirection(attack.direction)
```

Comment:
should this be `defense` as well?

Reply:
What about extracting the `MonsterSpell` configuration code into its own function so we can share it between `attacks` and `defenses`?

Reply:
nevermind--I thought these functions were part of your PR for a second.

Reply:
I'm glad you guys are checking!